hack/concat-yaml.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
(missed this comment when replying previously, sorry!)

This is for concatenating crds together, which is currently done in [deploy/crds/BUILD.bazel](https://github.com/jetstack/cert-manager/blob/master/deploy/crds/BUILD.bazel#L14-L19) (and it's done unsafely in bazel because it doesn't check for `%YAML` lines)

This is used in `bin/scratch/crds/all-crds.yaml` defined in `make/helm.mk` for now.

I'm not actually sure how if we could use `helm template` for that - but I'm sure there are better tools available!

Reply:
Same, re `hack/util`.